config/sync/views.view.divisions.yml

```diff
@@ -0,0 +1,207 @@
+uuid: 0499afec-9bb8-4d0c-a0bf-a0a6ac50e98c
+langcode: en
+status: true
+dependencies:
+  config:
+    - node.type.division
+  module:
+    - node
+    - user
+id: divisions
+label: Divisions
+module: views
+description: 'The image grid showing the different divisions on the About Us page.'
+tag: ''
+base_table: node_field_data
+base_field: nid
+display:
+  default:
+    display_plugin: default
+    id: default
+    display_title: Master
+    position: 0
+    display_options:
+      access:
+        type: perm
+        options:
+          perm: 'access content'
+      cache:
+        type: tag
+        options: {  }
+      query:
+        type: views_query
+        options:
+          disable_sql_rewrite: false
+          distinct: false
+          replica: false
+          query_comment: ''
+          query_tags: {  }
+      exposed_form:
+        type: basic
+        options:
+          submit_button: Apply
+          reset_button: false
+          reset_button_label: Reset
+          exposed_sorts_label: 'Sort by'
+          expose_sort_order: true
+          sort_asc_label: Asc
+          sort_desc_label: Desc
+      pager:
+        type: some
+        options:
+          items_per_page: 8
+          offset: 0
+      style:
+        type: grid
```

Comment:
I would try using the html list style, and apply the grid using css. The menu that exists at the moment is an HTML list, so you're more likely to get the same result with similar css code.